samples/highcharts/demo/venn-point-legend/demo.js

```diff
@@ -0,0 +1,53 @@
+Highcharts.chart('container', {
+    accessibility: {
+        point: {
+            descriptionFormatter: function (point) {
+                var intersection = point.sets.join(', '),
+                    name = point.name,
+                    ix = point.index + 1,
+                    val = point.value;
+                return ix + '. Intersection: ' + intersection + '. ' +
+                    (point.sets.length > 1 ? name + '. ' : '') + 'Value ' + val + '.';
+            }
+        }
+    },
+    series: [{
+        showInLegend: true,
+        legendType: 'point',
+        type: 'venn',
+        name: 'The Unattainable Triangle',
+        data: [{
+            sets: ['Good'],
+            value: 2,
+            name: 'Good'
+        }, {
+            sets: ['Fast'],
+            value: 2,
+            name: 'Fast'
+        }, {
+            sets: ['Cheap'],
+            value: 2,
+            name: 'Cheap'
+        }, {
+            sets: ['Good', 'Fast'],
+            value: 1,
+            name: 'More expensive'
+        }, {
+            sets: ['Good', 'Cheap'],
+            value: 1,
+            name: 'Will take time to deliver'
+        }, {
+            sets: ['Fast', 'Cheap'],
+            value: 1,
+            name: 'Not the best quality'
+        }, {
+            sets: ['Fast', 'Cheap', 'Good'],
+            value: 1,
+            name: 'They\'re dreaming'
+        }]
+    }],
+
+    title: {
```

Comment:
For a brief explanation of the sample, add a `subtitle` saying "A venn diagram with legend enabled"